__tests__/project.test.js

```diff
@@ -242,3 +242,41 @@ describe("PATCH /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("/api/comments/:comment_id", () => {
+  test("comment should be removed, and return an empty object for a body", () => {
+    return request(app)
+      .delete("/api/comments/1/")
+      .expect(204)
+      .then(({ body }) => {
+        expect(body).toEqual({});
```

Comment:
We dont need to expect the body to be an empty object here. With a status code of 204 this will always be the case so it doesnt have to be tested for